Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM eclipse-temurin:11 as builder
+# Multi stage build - https://docs.docker.com/engine/userguide/eng-image/multistage-build/
+
+# install maven and unzip
+RUN apt-get update && apt-get install -y --no-install-recommends maven unzip
+
+# prepare folders
+RUN mkdir /build && mkdir /target
+COPY ./ /build/
+
+# build and extract deegree
+RUN cd /build/ && \
+  mvn clean install -B -DskipTests && \
+  cp /build/deegree-ogcapi-webapp/deegree-ogcapi-webapp-postgres/target/deegree-ogcapi-webapp-postgres-*.war /build/deegree-ogcapi.war && \
+  unzip -o /build/deegree-ogcapi.war -d /target
+
+# add to image...
+FROM tomcat:9.0-jdk11-temurin-jammy
+
+ENV LANG en_US.UTF-8
+
+# add build info labels that can be set on build
+# see also https://github.com/opencontainers/image-spec/blob/master/annotations.md
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VCS_URL
+LABEL org.opencontainers.image.created=$BUILD_DATE \
+  org.opencontainers.image.source=$VCS_URL \
+  org.opencontainers.image.revision=$VCS_REF
+
+# tomcat port
+EXPOSE 8080
+
+# get dataset list as health check
+HEALTHCHECK \
```

Comment:
Not sure if this makes sense as a health check. Otherwise we can replace it with something else or just ditch it.

Reply:
The question here is, should the Tomcat healtcheck functionality here might be a better target ?
https://tomcat.apache.org/tomcat-9.0-doc/config/valve.html#Health_Check_Valve